app/components/elements/Dialog.js

```diff
@@ -24,13 +24,20 @@ export function DialogTitle(props) {
     ...dialogTitleProps
   } = props;
 
+  const ICON_WIDTH = 28;
+
   return (
     <Flex
       p={3}
-      sx={{ alignItems: 'center', justifyContent: 'space-between', borderBottom: props.divider ? borders.divider : 'unset', ...sx }}
+      sx={{ alignItems: 'center', borderBottom: props.divider ? borders.divider : 'unset', ...sx }}
```

Comment:
Based on the Figma doc, my understanding is that we are trying to align all modal styles, so I think having the following changes reflected on all modals is okay:
- Center Title
- Reduce Title Font Size